homeworks/09-chokidar/src/importer.ts

```diff
@@ -0,0 +1,37 @@
+import fs from "fs";
+const csv = require("csvtojson");
+const { EventEmitter } = require("events");
+
+class Importer {
+  eventEmitter;
+  constructor(eventEmitter: typeof EventEmitter) {
+    this.eventEmitter = eventEmitter;
+  }
+
+  listen() {
+    this.eventEmitter.on("dirwatcher:changed", (filePath: string) =>
+      csv()
+        .fromFile(filePath)
```

Comment:
If I add an image file to `data` folder then exception will be thrown here. I suggest filtering out other file types.